src/stores/ForeignStore.js

```diff
@@ -1,6 +1,6 @@
 import { action, observable } from 'mobx';
-import FOREIGN_ABI from '../abis/ForeignBridge.json';
-import ERC677_ABI from '../abis/ERC677.json';
+import { abi as FOREIGN_ABI } from '../submodules/poa-bridge-contracts/build/contracts/ForeignBridgeNativeToErc';
```

Comment:
Is this correct that `submodules/poa-bridge-contracts/build/contracts/ForeignBridgeNativeToErc` is specified without `.json`?

Reply:
Thanks for pointing this. In this case it's ok because we are using `create-react-app` that uses `webpack` which doesn't need the extension file to resolve the import. However I'll add the file extension `.json` for consistency since they are present on other imports